omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -774,8 +774,8 @@ def get_crop_region(self, panel):
         zoom = float(panel['zoom'])
         frame_w = panel['width']
         frame_h = panel['height']
-        dx = panel['dx']
-        dy = panel['dy']
+        dx = panel.get('dx', 0)
```

Comment:
since you have code after checking if ``if dx > 0`` (same for ``dy``) it will be better to introduce a default value ``default_value= 0``. So if later it changes it is only one place to adjust

Reply:
I can't see that we'd ever want a default offset that wasn't zero.
The default for these offsets is really set in the ```panel_model.js``` and it's only rarely (not reproducible) that it's not set here in the script (so we have to catch it just in case).

Reply:
Why 0 is set in ``get``, this is "discovered" further down in another check
having a default value makes that clear and explicit

Reply:
could introduce a variable with value 0 I think it will make the logic clearer